horizons/world/traderoute.py

```diff
@@ -31,6 +31,8 @@
 from horizons.util.worldobject import WorldObject
 from horizons.world.units.unitexeptions import MoveNotPossible
 
+def cmp(a, b):
```

Comment:
Was this function missing previously? Could you attach the related traceback to the issue / PR?

Reply:
I'm sorry, this was just found to be undefined in my own play of the game. (when I tried to setup a trade route, the game crashed, complaining that this function was not found)
- I didn't setup any issue, I just fixed it.

Reply:
`cmp` is a Python 2 function, that's probably why it was undefined if you were running the game with Python 3. Perhaps a remnant of refactoring from Python 2? If it's used only once in the entire code base, I think it would be better to replace the functionality internally instead :).

Reply:
It's used twice on line 134, not entirely sure the best method to clean this up.